src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public class Lottery {
+    public static Ball getRandomBall() {
+        return new Ball();
```

Comment:
here you must return created ball with a random color and random number, passing them in the constructor of the `Ball` class